indigo_app/templates/indigo_api/_work_filter_form.html

```diff
@@ -0,0 +1,57 @@
+{% load account %}
+
+
+<form method="GET" id="work-filter-form">
+  <div class="d-flex">
+
+    <div class="flex-grow-1 mr-2">
+        <input type="text" class="form-control filter-search" name="search_string" placeholder="Filter by title, year or number" value="">
+    </div>
+
+    <!-- TODO: Types filter -->
+    <div class="mr-2">
+        <select class="form-control filter-subtype" name="subtype">
+          <option value="all">All types</option>
+          <option value="-">Acts</option>
+          <option value="by-law">By-law</option>
+          <option value="gn">Government Notice</option>
+          <option value="p">Proclamation</option>
+          <option value="si">Statutory Instrument</option>
+        </select>
+      </div>    
+
+    <!-- Stubs filter -->
+    <div class="mr-2">
+        <select class="selectpicker filter-stub" name="stub" data-style="btn-outline-secondary" data-width="auto">
+          <option value="excl">Exclude stubs</option>
+          <option value="only">Only stubs</option>
+          <option value="all">Everything</option>
+        </select>
+      </div>      
+
+    <!-- Status filter -->
+    <div class="mr-2">
+      <select class="selectpicker notooltip" multiple name="status" data-style="btn-outline-secondary">
```

Comment:
Hardcoding the options for these selects is fine. Don't forget to render which option is currently active.